ash-window/Cargo.toml

```diff
@@ -15,13 +15,13 @@ workspace = ".."
 
 [dependencies]
 ash = { path = "../ash", version = "0.35", default-features = false }
-raw-window-handle = "0.3"
+raw-window-handle = "0.4.2"
 
 [target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies]
-raw-window-metal = "0.1"
+raw-window-metal = "0.2.0"
 
 [dev-dependencies]
-winit = "0.19.4"
+winit = "0.26.1"
```

Comment:
This probably needs additional changes, no?

Reply:
cargo check --workspace seemed to work

Reply:
`--workspace` only checks the crates (binaries, libraries) in the `workspace` (and it seems to be the default when the root `Cargo.toml` only defines the workspace but doesn't contain any crate of its own).

You should however also `check` the examples with `--examples`, or more generically `--all-targets`.